101-app-service-create-managed-certificate-apex/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,80 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "domainName": {
+        "type": "string",
+        "metadata" : {
```

Comment:
format the json, e.g. SHIFT+ALT+F in VS Code

Reply:
done